app/models/user.rb

```diff
@@ -141,14 +143,13 @@ def set_defaults
     @meeting_notify.nil? && @comment_notify = true
   end
 
-  OAUTH_TOKEN_URL = 'https://accounts.google.com/o/oauth2/token'
-
   def update_access_token
     refresh_token_params = { 'refresh_token' => refresh_token,
                              'client_id'     => nil,
                              'client_secret' => nil,
                              'grant_type'    => 'refresh_token' }
-    response = Net::HTTP.post_form(User::OAUTH_TOKEN_URL, refresh_token_params)
+    uri = URI.parse(User::OAUTH_TOKEN_URL)
+    response = Net::HTTP.post_form(uri, refresh_token_params)
```

Comment:
This method never succeeds. We might not have seen the errors before because of the rescue from Standard error. I am getting 

```{"error"=>"invalid_request",
 "error_description"=>"Could not determine client ID from request."}```

Possibly because we don't have `client_id` and `client_secret`. @julianguyen do we have these credentials for testing?

Reply:
Good find :O I'll set up values for GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET in our CircleCI environment variables.

Reply:
Just set it up, try running again 🤞

Reply:
So, can I change 

```
'client_id'     => ENV['GOOGLE_CLIENT_ID'],
'client_secret' => ENV['GOOGLE_CLIENT_SECRET']
```